tests/functional/helper.js

```diff
@@ -160,6 +160,15 @@ module.exports = {
         `Command failed: ${json.error}`);
   },
 
+  async waitForPropertyAndClickOnElement(id, property) {    
+    await this.waitForElementProperty(property ? property : id, 'visible', 'true');
```

Comment:
I don't understand the logic here. The first argument should be the element, but you are passing the property. Can you tell me how do you want to use this?

Reply:
Plus it seems that a second argument is not used in this code.